lib/dialects/mysql/index.js

```diff
@@ -1,16 +1,50 @@
 'use strict';
 
 const _ = require('lodash');
-const AbstractDialect = require('../abstract');
+const { AbstractDialect } = require('../abstract');
 const ConnectionManager = require('./connection-manager');
 const Query = require('./query');
 const QueryGenerator = require('./query-generator');
 const DataTypes = require('../../data-types').mysql;
 const { MySQLQueryInterface } = require('./query-interface');
 
 class MysqlDialect extends AbstractDialect {
+  static supports = _.merge(
+    _.cloneDeep(AbstractDialect.supports),
+    {
+      'VALUES ()': true,
+      'LIMIT ON UPDATE': true,
+      lock: true,
+      forShare: 'LOCK IN SHARE MODE',
+      settingIsolationLevelDuringTransaction: false,
+      inserts: {
+        ignoreDuplicates: ' IGNORE',
+        updateOnDuplicate: ' ON DUPLICATE KEY UPDATE'
+      },
+      index: {
+        collate: false,
+        length: true,
+        parser: true,
+        type: true,
+        using: 1
+      },
+      constraints: {
+        dropConstraint: false,
+        check: false
+      },
+      indexViaAlter: true,
+      indexHints: true,
+      NUMERIC: true,
+      GEOMETRY: true,
+      JSON: true,
+      REGEXP: true,
+      _test_: 2
+    }
+  );
+
   constructor(sequelize) {
     super();
+
```

Comment:
I don't think this empty line is consistent with the other dialects

Reply:
Removed!